src/components/ApplyFlow.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCheckCircle } from '@fortawesome/free-solid-svg-icons';
+
+export default function ApplyFlow(props) {
+  return (
+    <div className="field">
+      <p className="control has-icons-left">
+
+        <select
+          className="select"
+          name="apply_flow"
+          id="apply_flow"
+          value={props.applyFlow}
```

Comment:
assuming the above props declaration change this becomes:

`value={applyFlow}`

Reply:
Yep, updated this too.